ironic.conf

```diff
@@ -10,11 +10,11 @@ enabled_deploy_interfaces = direct,fake
 # NOTE(dtantsur): when changing this, make sure to update the driver
 # dependencies in Dockerfile.
 enabled_hardware_types = ipmi,idrac,irmc,fake-hardware,redfish,ibmc
-enabled_inspect_interfaces = inspector,idrac,irmc,fake,redfish
-enabled_management_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,ibmc
-enabled_power_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,ibmc
-enabled_raid_interfaces = no-raid,irmc,agent,fake
-enabled_vendor_interfaces = ipmitool,no-vendor,idrac,fake,ibmc
+enabled_inspect_interfaces = inspector,idrac,irmc,fake,redfish,idrac-wsman
+enabled_management_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,ibmc,idrac-wsman
+enabled_power_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,ibmc,idrac-wsman
+enabled_raid_interfaces = no-raid,irmc,agent,fake,idrac-wsman
+enabled_vendor_interfaces = ipmitool,no-vendor,idrac,fake,ibmc,idrac-wsman
```

Comment:
We should have raised it earlier, but why do we need vendor interfaces? They're not exposed in BMO.

Reply:
We got the PR #146 approved for RAID interface but I accidentally removed the parent repository.
Removed vendor_interface.

Reply:
Removed idrac-wsman interface for vendor interface.